src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -1,6 +1,13 @@
 <div class="row">
   <div class="col-sm-12 col-lg-12">
     <div>
+      <span *ngIf="!rgwModuleStatus">
+      <cd-alert-panel   type="info"
+                        i18n>You need to enable the rgw module to access the import/export feature.
+                        <button class="btn btn-primary"
+                                type="button"
+                                (click)="enableRgwModule()">
+                                Enable rgw Module</button></cd-alert-panel><br></span>
```

Comment:
This alert doesn't affect the whole page, only some certain actions inside this page like Import/Export functionality. So isn't it better to keep it as part of that form only rather than the entire page?

Reply:
This can be discussed, I don't agree with this.